util/json.go

```diff
@@ -0,0 +1,40 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
Fair point. I agree, let's not let it become "grab-all package". Changed to jsonutil.